_amenities/klahanie-neighborhood-patrol.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: default 
+title: Klahanie Neighborhood Patrol
+date: 2019-06-10 03:42
+author: Klahanie
+categories: []
+show-title: true
+---
+<div class="float-right col-md-4">
+  <a href="{{site.url}}vacation-house-check.html">Vacation House Check Form</a>
+  <img src="{{site.url}}images/canine-patrol.jpg" class="img-thumbnail">
+  <img src="{{site.url}}images/KNP-2.jpg" class="img-thumbnail">
+  <img src="{{site.url}}images/shoreline-vehicle-2.jpg" class="img-thumbnail">
+</div>
+
+
+<p class="lead">Providing unduplicated levels of neighborhood security</p>
+<em>by Sgt. Steve Lysaght</em>
+
+For the past seven years it has been my pleasure to be the Director of your Klahanie Neighborhood Patrol (KNP).  I am also a Sergeant with the King County Sheriff’s Office.
+
+Each year the KNP program has continued to grow and evolve in providing you with additional police-related services.  With the turnover in residents, I thought it best to re-introduce ourselves to give each of you a better understanding of the enhanced security you receive.
+
+Currently, your KNP is comprised of 23 fully-commissioned Deputies, Detectives and Sergeants. Each of us is employed by the King County Sheriff’s Office.  Perhaps you have noticed patrol cars with the names of other communities driving through your neighborhood?  Those officers are not lost, nor do they live here; instead, they are on patrol protecting you and your family. Each officer has permission to use their patrol vehicles while working in Klahanie. The time spent here is in addition to their regular jobs—this is referred to as “off-duty” work.
+
+<strong>Police, Sheriff or Both?</strong>
+
+The City of Sammamish contracts with the Sheriff’s office to provide deputies to staff the Police Department. These officers are tasked with patrolling the incorporated area of Sammamish, which includes Klahanie.  However, Sammamish is a large geographical area with a rapidly growing population.  For this reason, your Klahanie Homeowner’s Association created the KNP, whose sole responsibility is to provide enhanced police services to ensure the public safety of the Klahanie community.  The KNP have all the legal powers to ticket, arrest or apprehend anyone breaking the law.
+
+While in-service, the KNP Deputies are actively monitoring all emergency and non-emergency radio traffic occurring both inside and outside of Klahanie.  The KNP is staffed daily by three or more Deputies, on staggered shifts, to ensure a cost-effective use of the Klahanie Public Safety budget.  We drive both marked and unmarked police vehicles, and work in uniformed and plain-clothes capacities.  Our staff currently includes five sergeants, four detectives, two SWAT team members, and two K9s, each with a depth of training and experience.  Our daily patrols are usually staffed during morning and early afternoon hours, when burglaries are most likely to occur, as well as late evening and overnight when vehicle prowls are most common.

```

Comment:
"the KNP deputies"
"three or more deputies"
Is "Klahanie Public Safety" an actual group or document? If not, "Klahanie public safety budget."